package.json

```diff
@@ -102,6 +106,11 @@
   },
   "dependencies": {
     "postcss": "^8.3.6",
-    "react-table": "^7.7.0"
+    "react-infinite-scroll-component": "^6.1.0",
+    "react-table": "^7.7.0",
+    "react-table-sticky": "^1.1.3",
```

Comment:
is it necessary to use this library?

Reply:
Yes for sticky header we need react-table-sticky library